<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupDoctors">
      <property name="title">
       <string>Доктора</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayoutDoctorsActions">
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButtonDoctorsAdd">
           <property name="text">
            <string>Добавить</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButtonDoctorsDelete">
           <property name="text">
            <string>Удалить</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButtonDoctorsSearch">
           <property name="text">
            <string>Поиск</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButtonDoctorsClearSearch">
           <property name="text">
            <string>Очистить поиск</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableDoctors">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxPatients">
      <property name="title">
       <string>Пациенты</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tablePatients"/>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayoutPatientsActions">
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButtonPatientsAdd">
           <property name="text">
            <string>Добавить</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButtonPatientsDelete">
           <property name="text">
            <string>Удалить</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButtonPatientsSeatch">
           <property name="text">
            <string>Поиск</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButtonPatientsClearSearch">
           <property name="text">
            <string>Очистить поиск</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxAppointments">
      <property name="title">
       <string>Записи</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableAppointments"/>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayoutAppointmentsActions">
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButtonAppointmentsDelete">
           <property name="text">
            <string>Удалить</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButtonAppointmentsAdd">
           <property name="text">
            <string>Добавить</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButtonAppointmentsSearch">
           <property name="text">
            <string>Поиск</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButtonAppointmentsClearSearch">
           <property name="text">
            <string>Очистить поиск</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>842</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionFileOpen"/>
    <addaction name="actionFileSave"/>
    <addaction name="separator"/>
    <addaction name="actionFileExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFileOpen">
   <property name="text">
    <string>Открыть...</string>
   </property>
  </action>
  <action name="actionFileSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="actionFileExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
